test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -35,10 +36,11 @@
 #include "model/commands/set_quorum.hpp"
 #include "model/commands/subtract_asset_quantity.hpp"
 #include "model/commands/transfer_asset.hpp"
-#include "model/execution/command_executor_factory.hpp"
 #include "model/permissions.hpp"
 #include "module/irohad/ametsuchi/ametsuchi_mocks.hpp"
 
+#include "backend/protobuf/from_old_model.hpp"
```

Comment:
I think this test can be reworked with shared_model so there is no need in conversion from old model.